src/components/reference/_supported-languages-table.mdx

```diff
@@ -14,8 +14,8 @@
 | PHP        | GA | Beta | -- |                   
```

Comment:
I don't think `JSON` has any special "cross-function" support.

Reply:
Makes sense. Removed beta in JSON. Thank you for the catch.